packages/@aws-cdk/custom-resource-handlers/lib/custom-resources-framework/classes.ts

```diff
@@ -373,4 +356,30 @@ export abstract class HandlerFrameworkClass extends ClassType {
     const superInitializerArgs: Expression[] = [scope, id, props.superProps];
     init.addBody(new SuperInitializer(...superInitializerArgs));
   }
+
+  private buildRuntimeProperty(scope: HandlerFrameworkModule, options: BuildRuntimePropertyOptions = {}) {
+    const { runtime, isCustomResourceProvider, isEvalNodejsProvider } = options;
+
+    if (runtime) {
+      return isCustomResourceProvider ? expr.lit(runtime) : expr.directCode(toLambdaRuntime(runtime));
+    }
+
+    if (isCustomResourceProvider) {
+      scope.registerImport(CORE_MODULE, {
+        targets: [CORE_MODULE.determineLatestNodeRuntimeName],
+        fromLocation: scope.isCoreInternal
+          ? '../../custom-resource-provider'
```

Comment:
I'm seeing paths hardcoded a lot. Can we turn most of these into a const?

Reply:
We could unintentionally break ourselves if we ever move packages around.

Reply:
Agreed. Will move these into constants. Probably as part of the defined modules.